src/server.rs

```diff
@@ -106,7 +106,7 @@ impl Server {
 
         thread::spawn(move || {
             actix_web::rt::System::new().block_on(async move {
-                debug!("The number of workers are {}", workers.clone());
```

Comment:
shouldn't this be `are` only? We can have multiple workers at the moment.

Reply:
Well English isn't my first language, but I would have said `is` here, because we refer to "The number" which is singular?

Reply:
No no. Since the number of workers are more than one, it should be `are`